.gitignore

```diff
@@ -4,6 +4,9 @@ _targets
 #All model output
 */out/*
 
+#Other local files
+archive/*
```

Comment:
What is in the archive? Just curious. I guess this must be your local files because it's not showing up when running the pipeline

Reply:
Yep, just local scripts and notes that I didn't want git to track